app/controllers/jobseekers/email_preferences_controller.rb

```diff
@@ -0,0 +1,25 @@
+module Jobseekers
+  class EmailPreferencesController < BaseController
+    before_action :set_model
+
+    def edit; end
+
+    def update
+      if @model.update(email_preferences_params)
```

Comment:
[Minor] I would usually go for @jobseeker or something a bit more descriptive here.